Make goats great again v2.0

Zert0X opened the pull request:
Возродил #4211
Все права принадлежат @MrPersival от награды за иссуй отказываюсь  

1. Добавлен шанс сбить с ног.
2. Если дать козлу перец, он впадет в рейджмод, в который, правда, и так может впасть время от времени. Задержка есть и так, за её время можно убежать.
3. Словарный запас увеличен.
4. Теперь с козла выпадает шкура при разделывании. В шкуре можно прорезать дырку для головы, и носить.
5. Доение уже было, не трогал.
6. Друиду в Monster Manual добавлен новый фамильяр - козел. Сбивает с ног с помощью magic missle, не сбивает при атаках.
<details>
<summary>Чейнджлог</summary>

```yml
🆑 MrPersival
rscadd: Козлы теперь могу сбивать с ног.
rscadd: Если дать козлу перец то произойдёт интересное.
rscadd: Козлы выучили пару новых слов.
rscadd: Добавлена шкура козла
rscadd: Козёл теперь есть как фамильяр у мага.
/🆑
```

</details>
close #1426

- [x] Pull Request полностью завершен, мне не нужна помощь чтобы его закончить.
- [x] Я внимательно прочитал все свои изменения и багов в них не нашел.
- [x] Я запускал сервер со своими изменениями локально и все протестировал.
- [x] Я ознакомился c [Guide to Contribute](https://github.com/ChaoticOnyx/OnyxBay/blob/dev/docs/contributing.md).

Zert0X committed:
Make goats great again

Возродил #4211

Zert0X committed:
codestyle cop

Zert0X committed:
Cow CODESTYLE

Zert0X committed:
Merge branch 'ChaoticOnyx:dev' into make-goats-great-again

Zert0X committed:
Merge branch 'make-goats-great-again' of https://github.com/Zert0X/OnyxBay into make-goats-great-again

Doster-d reviewed (approved):
Вроде неплохо, хорошая работа, Ловла

Zert0X commented:
та погоди куда, спрайтов для альтслимов ещё нет:kek:

Doster-d commented:
Спрайты для лохов, русские создают картинку в голове из описание.

Doster-d reviewed (changes requested):
Спайты.

Commit:
tweak(Frontier): fixes assorted stuff

PR #5678
- Добавлены **RPD** и **pipe painter** на карту "Frontier".
- Добавлен ресливер в комнату медицинского отсека с крио капсулой на карту "Frontier". 
- В ксеноботанике на карте "Frontier" CondiMaster был заменен на ChemMaster.
- Убрана лампа, которая мешала сканировать артефакты в ксеноархеологии на карте "Frontier".
- Проведены провода к эмиттерам в ксеноархеологии на карте "Frontier".
- По многочисленным заявкам исправлено странное соединение труб в атмосферном отсеке на карте "Frontier".
- Заделаны дыры в машинном отделении на карте "Frontier".
- Исправлен инжектор в камере сжигания.

TobyThorne committed:
tweak(light fixtures): broken lights can be removed with crowbars

PR #5661
- Теперь разбитые лампочки можно вытаскивать из светильников с помощью монтировки.

igorsaux committed:
Changelog 21/06/21 (#5686)

igorsaux committed:
fix(build): removes sounds sticked into to_chat

PR #5697

TatarTheBuilder committed:
tweak(Frontier): fixes bugs, adds features

PR #5696 
- Исправлен кабель в отделе ксеноархеологии на Фронтире.
- Исправлены кнопки бластдуров у сингулярности на Фронтире.
- Добавлена новая локация в северо-западных техах Фронтира.

igorsaux committed:
tweak(chat): fixes DOOC color, changes Help appearance

PR #5694

Commit:
tweak(mecha): trackers no longer get spawned on their own

PR #5680

TapKo4eB committed:
fix(autolathe): material loading animation fix

PR #5710 
- Исправлены анимации пополнения автолата стеклом и при открытой крышке.

TobyThorne committed:
tweak(xenomorphs): undressing comes real

PR #5658 
- Ксеноморфы теперь могут раздевать людей, но с ограничениями.
- Лицехваты теперь скидывают не только шлемы и маски, но и предметы на ушах.

Zert0X committed:
Slimalt, slim_M sprites

Ну вроде как да

Zert0X commented:
Призываю своих собратьев по чулкокодерскому ордену!

DarkHunter7756 commented:
> Призываю своих собратьев по чулкокодерскому ордену!

Тебе никогда не получится замержить козлов. Иссуй проклят. Этот иссуй проложен козлами и козлы хранят его. Иссуй проклят.

Zert0X commented:
> 
> 
> > Призываю своих собратьев по чулкокодерскому ордену!
> 
> Тебе никогда не получится замержить козлов. Иссуй проклят. Этот иссуй проложен козлами и козлы хранят его. Иссуй проклят.

Это пр :kek:
Кому нужен тот иссуй если есть ПР!

Doster-d commented:
![Screenshot_20210624-213334-01.jpeg](https://user-images.githubusercontent.com/63081538/123315021-ea8ba200-d533-11eb-96ed-6ef1e4a71f53.jpeg)
Видимо это и есть та проблема с козлами

Zert0X commented:
Я твой рот труба шаталь
![H1fj](https://user-images.githubusercontent.com/42303535/123315849-da27f700-d534-11eb-8b70-cc9103af3a73.gif)

Zert0X committed:
items.dmi

Я твой рот труба шаталь

kemperhun committed:
fix(viruses): fixes few bugs

PR #5734 
- Исправлена ошибка, из-за которой вирус не передавался дыхательным путем между людьми.
- Two Percent Syndrome больше не гибает людей.
- При мутациях у новых эффектов теперь корректная стадия.
- Исправлена ошибка, из-за которой мутировавший внутри человека вирус был неизлечим.

Commit:
tweak(Exodus): fixes assorted stuff, adds new shuttle chairs subtype

PR #5739 
- Удалена документация психоскопа с Исхода.
- Обновлен эвакуационный шаттл "Исхода" - удален стол из-под консоли, заменены некоторые клетки пола.
- Добавлен органфиксер воксам и нюкерам.
- Обновлен шаттл шахтеров - стулья заменены на новые кресла.
- Обновлен шаттл синдиката - стулья заменены на кресла, исправлена планировка кабины пилота.

TatarTheBuilder committed:
tweak(Frontier): adds new lightbulb boxes, fixes some bugs

PR #5735
- На Фронтире теперь лежат коробки с новыми лампами.
- Изменена мусорка в ксенобио на Фронтире.

No-Tips committed:
tweak(killer tomatoes): tweaks movespeed

PR #5708

RdEErAqUa committed:
tweak(tajaran): tajaran overheat while wearing spacesuits

PR #5421

TatarTheBuilder committed:
tweak(light fixtures): replaces hanging lantern with old light fixtures

PR #5715

Doster-d committed:
tweak(DBBan): shows the banned player a notification

PR #5766

TobyThorne committed:
tweak(flashbangs): reworks flashbangs

PR #5719 
- Нерфобафф светошумовых гранат. Меньше стана и нокдауна, больше ослепления и оглушения.

TobyThorne committed:
tweak(uplink): changes nuclear ops' TC amount to 15, some prices go up, reworks assault rifles

PR #5769 
- Переработана штурмовая винтовка 5.56мм. Нарисованы новые спрайты. Название изменено с STS-35 на AS-75. Вместимость магазина увеличена с 20 до 25 патронов. Урон пулек 5.56мм увеличен с 25 до 27.5.
- Запас кристаллов в аплинке нюкеров повышен с 12 до 15 ТК.
- Повышены цены на некоторое оружие нюкеров чтобы скомпенсировать увеличенный запас.
- В аплинк нюкеров добавлен пулемёт, L6 SAW (за 12 ТК) и коробка на 60 патронов для него (за 3 ТК).
- Подкорректирован флешдарк. Теперь темнота от него накладывается сильнее и не отдаёт синим.
- Исправлена невозможность покупки хамелеон-бомбы в аплинке.
- Очевиднее названы некоторые позиции патронов в аплинке (добавлены приписки "Magazine", "Speedloader", etc).

NefrNerf committed:
fix(adminpaper): fixes user access to admin fax

PR #5669

igorsaux committed:
Changelog 26/06/21 (#5772)

RdEErAqUa committed:
tweak(tajaran): tajaran overheat while wearing spacesuits, but now without damage

PR #5782

igorsaux committed:
fix(build): fixes 3 runtimes

PR #5801

fixes #5800
fixes #5799
fixes #5797

igorsaux committed:
tweak(vending machines): changes NanoUI to TGUI

PR #5713 
- Редизайн интерфейса торговых автоматов.

Shirlineyn committed:
fix(build): fixes assorted borg-related bugs, allows drag-n-dropping ppl into resleevers, allows traitorborgs to emag themselves

PR #4820 
- Cервис-грипперы теперь могут в кухонное оборудование.
- Грипперы теперь могут вставлять батарейки в PSUшки.
- Дропперы теперь нельзя перемолоть в гриндере.
- В ресливер теперь можно класть мобов драг'н'дропом.
- Дроны теперь могут в надувные стены.
- Киборги-триторы могут емагнуть себя сами, используя соответствующий верб.

TobyThorne committed:
fix(icons): centers a number of icons

PR #5808

igorsaux committed:
fix(build): adds icon2base64 for better icon-HTML handling

PR #5811

TapKo4eB committed:
fix(water_ballon): w_size approximation & fulfillment from dispensers

PR #5709 
- Водяные шарики теперь приобрели ожидаемый малый размер и научились менять его при наполнении.
- Водяные шарики теперь можно наполнять не только из водяных баков.

TobyThorne committed:
fix(medicine): fixes carotid artery typo

PR #5806

igorsaux committed:
tweak(chat): adds graphic emoji support

PR #5819

No-Tips committed:
fix(corgi): allows corgies to wear hats... again

PR #5698 
- Корги снова могут носить шапки.

TobyThorne committed:
tweak(events): reduces viruses' weight, makes some events non-oneshot

PR #5818 
- Уменьшен вес всех трёх уровней вирусов: Space Cold Outbreak (20 -> 15 за медика), Viral Infection (40 -> 20 за медика), Plague Infection (35 -> 20 за медика). Plague Infection теперь одноразовый ивент.
- Space Vines, Carp Migration, Carp School, Appendicitis, Lost Carp - больше не одноразовые ивенты.
- Вес Appendicitis снижен (15 -> 3 за медика), ибо теперь может произойти больше одного раза за раунд.
- Вес Wormholes увеличен (10 по умолчанию -> 10 по умолчанию + 1 за каждого игрока).

NefrNerf committed:
fix(wizard): teleporting from cryo no longer produces cringe

PR #5750 
- Добавлена проверка на нахождении в криокапсуле, если маг кастует из нее, то его сначала оттуда выкинет, и после этого произойдет блинк/телепорт.

TobyThorne committed:
fix(build): allows the build to compile

PR #5820 
- Исправлена ошибка с инклюдами из одного из ПРов.

igorsaux committed:
Changelog 29/06/21 (#5821)

Artemonim committed:
fix(viruses): Runtime in helpers.dm, 92: Cannot read null.name

PR #5822

Epicus7 committed:
feat(docs): add "as anything()" section to contributing.dm

TobyThorne committed:
tweak(backpacks): renews medical, sci and hydro backpacks worn icons

PR #5840 
-  Обновлены спрайты некоторых рюкзаков (у медиков, химиков, вирусологов и учёных) на спине. Теперь они выглядят соответственно спрайтам самих рюкзаков.

igorsaux committed:
fix(vending.dm): Runtime in vending.dm, 188: Cannot execute null.GetIdCard().

PR #5836

NefrNerf committed:
fix(mimics): defaults their obj type to crate 

PR #5841 
- Заспавненные мимики теперь по умолчанию будут ящиками, так что больше никаких бессмертных ебак.

Artemonim committed:
tweak(consoles): engineering consoles have nanoword preinstalled

PR #5827

NefrNerf committed:
fix(wizard): fixes knock and contract skills

PR #5848 
- Заклинание "Knock" открывает обесточенные шлюзы.
- Исправлены заклинания взаимодействия мага с учениками.
- Заклинание "Return to Master" телепортирует ученика к магу, а не наоборот.

TobyThorne committed:
fix(stun gloves): fixes broken initialize

PR #5865 
- Стан-перчатки инициализировались через жопу.
- Заспавненные/намаппленные стан-перчатки теперь по умолчанию выбирают базой рабочие перчатки. До этого - получалось поломанное нечто.
- Попытка обрезать/обмотать проводами перчатки-хамелеоны теперь выдаёт очевидное сообщение, как у боксёрских перчаток.

igorsaux committed:
tweak(R&D machinery): changes plain browser to TGUI

PR #5859

TermService commented on code/modules/clothing/suits/miscellaneous.dm:
```diff
@@ -609,3 +609,26 @@
+	body_parts_covered = UPPER_TORSO|LOWER_TORSO
+	cold_protection = UPPER_TORSO|LOWER_TORSO
+	min_cold_protection_temperature = ARMOR_MIN_COLD_PROTECTION_TEMPERATURE
+	armor = list(melee = 25, bullet = 10, laser = 0, energy = 40, bomb = 0, bio = 10, rad = 0)
```
energy=40
Защита уровня скафандров и некоторых РИГов ЕРТ, непойдет, но если геймдизайнеры не против то ладно

Zert0X replied:
Отказано, иди читай оригинальный ПР, там сами гейдизы выставили такое значение

TermService commented on code/modules/clothing/suits/miscellaneous.dm:
```diff
@@ -609,3 +609,26 @@
+	cold_protection = HEAD
+	flags_inv = HIDEEARS | BLOCKHAIR
+	min_cold_protection_temperature = ARMOR_MIN_COLD_PROTECTION_TEMPERATURE
+	armor = list(melee = 25, bullet = 10, laser = 0, energy = 40, bomb = 0, bio = 10, rad = 0)
```
тоже самое что и выше

TobyThorne committed:
tweak(sfx): cuts a delay before the actual sound in smash.ogg

PR #5883

TobyThorne committed:
fix(build): fixes skulls dupe, some weird xenobehavior, reentering qdeleted bodies, etc

PR #5891 
- Исправлена возможность бесконечно спавнить черепа и ошмётки с помощью голов.
- Из голов дион и ксеноморфов больше нельзя доставать человеческие черепа.
- Добавлена задержка при ударах по мехам.
- Грудоломы теперь появляются через секунду после разрывания человека. Чтобы их не убивало летящими запчастями тел в тесных помещениях.
- При использовании Re-Enter Corpse добавлена проверка на нахождение тела в нуллспейсе. Поскольку сейчас практически никакие мобы нормально не куделятся, попытка вернуться в такого моба приводила ко всяким паранормальным явлениям вроде появления в виде мозга/хэдкраба-зомби в космосе/на астероиде/в жопе админа.
- Исправлена возможность появления у королевы ксеноморфов нейролейса.
- Исправлено превращение дрона в грудолома вместо королевы при использовании Evolve, если дрон появился не "натуральным" путём, а с помощью кривого использования админских кнопок.
- Чутка ослабил петлю на шее у GC, прибравшись в Destroy() у мобов. Большая часть мобов всё равно ещё где-то как-то референсится, но по крайней мере /mob/living/simple_animal уже нормально удаляются, что радует.
- Теперь при получении роли ксеноморфа будут выводиться разные сообщения в зависимости от того, является ли моб грудоломом или ксеноморфом.

TobyThorne committed:
fix(ammunition): fixes some missing ammo icons

PR #5881

TobyThorne committed:
tweak(vending machines): makes UI buttons thinner

PR #5867

igorsaux committed:
tweak(fonts): removes Shage

PR #5871 
...whatever it was

Epicus7 committed:
tweak(storyteller): adds more roles to storyteller

PR #5500 
- Добавляем больше ролей в сторителлера. Поправил рантайм при переключении вкладок панели управления сторителлером.
- При спавне борера, мага и ниндзи сторителлер опрашивает гостов и игроков в лобби с соответствующим префенсом, хотят ли они быть ролью. Если игрок отклоняет или не отвечает в течение 10 секунд, то опрашивается следующий игрок.
- Нужно еще допилить борерам перенос тушки КУДА-ТО - пока не придумал где их спавнить можно.

TobyThorne committed:
fix(floor): fixes floor tiles dupe

PR #5900 
- Исправлена возможность дюпать плитку, отдирая её с тайла, под которым есть стена.

TobyThorne committed:
fix(tape recorder): fixes broken icon_states

PR #5906 
- Диктофоны больше не невидимые.

Doster-d committed:
fix(build): items can no longer be thrown out of vents, banned players see some info

PR #5903 
- Из труб больше нельзя бросать предметы.
- Теперь при заходе с баном, помимо причины, выдается ссылка на то, куда писать, чтобы подать апелляцию.

igorsaux committed:
Changelog 04/07/21 (#5920)

TobyThorne committed:
fix(detective's guncase): fixes ammo description

PR #5949

Gr13v0us committed:
fix(loadout): makes P&T lighter clickable

PR #5934 
- Зажигалка "P&T" теперь нажимается в лодауте.

igorsaux committed:
fix(tgui): Optimization of RnD UI and some fixes

igorsaux committed:
Changelog 06/07/21 (#5961)

Gr13v0us committed:
tweak(loadout): removes IPC-only items

PR #5943

igorsaux committed:
tweak(R&D machinery): returns save design button

PR #5970

igorsaux committed:
tweak(arcade): changes plain browser to TGUI

PR #5967

NefrNerf committed:
tweak(crates): crates now use assigned material when deconstructed

PR #5689
- Ящики получили переменную для материала (используется при разборе).

Oubifx committed:
fix(twohanded): fixes two-handed weapons worn icons

PR #5771 
- Исправлено отображение двуручных видов оружия на спине.

TapKo4eB committed:
tweak(antags): roles' min_age_req now use cfg values

PR #5895

Zert0X committed:
fix(input): fixes central command reports and announces input fields

PR #5987 
- Окошки инпута для аннонсов и сообщений ЦК стали большими.

Gr13v0us committed:
tweak(syndicate toolbox): improves storage space

PR #5954 
- Увеличена вместительность синдикатовских тулобоксов.

Doster-d committed:
tweak(organs): organs now can be cooked

PR #5430 
- Можно готовить органы в кухонных устройствах, можно освежёвывать конечности и удобно их поедать.

Doster-d committed:
Changelog 07/07/21 (#5991)

igorsaux committed:
tweak(floor painter): implements TGUI

PR #5994

Gr13v0us committed:
tweak(matter decompiler): matches can be recycled now

PR #5938
- Matter decompiler теперь может перерабатывать спички.

No-Tips committed:
fix(metroid): Fixes adamantine's metroid sprite

igorsaux committed:
fix(R&D machinery): makes rnd console be more optimized

igorsaux committed:
Del(overmap): remove overmap

PR #5921
- Удалена система овермапов и всё что с ней связанно.

Doster-d committed:
feat(subsystems): explosion and virus subsystem update

PR #6008
- портирование системы взрывов с ТГ;
- виры теперь находятся в правильном приоритете.

igorsaux committed:
Генерация чейнджлога на 12.07.2021 (#6049)

Commit:
tweak(virusdish): new sprites for "virus dish".

igorsaux committed:
purge(code): remove a bunch of unused code

igorsaux committed:
feat(logging): adds story and integrated circuits logging

Doster-d committed:
fix(changeling): fixes changeling reagent mutation

PR #5989 
- При становлении генокрадом через генкореагент космонавтики больше не получают задания генокрадов.
- Космонавтики, ставшие генокрадами через генкореагенты или способность Division теперь показываются в списке антагонистов в конце раунда.

Doster-d committed:
fix(grab): adds more checks into processing proc

PR #6088 
- Добавлено больше проверок в процессинг грабов;

TobyThorne committed:
fix(prosthesis): fixes absent slim male chest prosthetic icons

PR $6089

Zert0X committed:
tweak(build): removes ban_everywhere option

PR #5939

Epicus7 committed:
fix(antagonist): fix update_antag_mob

PR #6097 
- Исправлен спавн воксов-рейдеров на станции вместо их собственной локации.

TobyThorne committed:
fix(antagonist): fixes xenomorphs and borers spawn

PR #6100 
- Бореры больше не наследуются от ксеноморфов;
- Исправлены возможные проблемы со спавном ксеноморфов;

TobyThorne committed:
feat(custom items): adds AmiClerick's custom labcoat

PR #6103 
- Добавлена кастомка для AmiClerick.

Oubifx committed:
fix(viruses): fixes antibodies mixing

PR #5953 
- Исправлено смешивание антител. Теперь их часть не пропадает.

Oubifx committed:
tweak(eswords): swords emmit light when active

PR #5993 
- Включенные энергетические мечи и кинжалы теперь светятся во включенном состоянии.

Nod404 committed:
fix(candies): adds "candy" state, changes default state

PR #6121

TobyThorne committed:
tweak(projectiles): reduces stunshells' penetraction modifier

PR #6107

TobyThorne committed:
tweak(snacks): removes cannabar, adds hemp2gen

PR #6125

igorsaux committed:
tweak(vending machines): removes flying burgers

PR #6129 
- Убраны летающие бургеры с фона UI вендоматов
- :-(

Zert0X committed:
(tweaks): vampire skills rebalance

* Vamp can see in darkness

* Larger text in chat

* Some abilities tweaks

* power use cost to var && some spaces

Doster-d committed:
tweak(portal): move all objects from portal to loc

* tweak(portal) move all objects from portal loc

* add wait timer

* update checks

TobyThorne committed:
fix(closet): can be repaired after hack

igorsaux committed:
fix(runtime): respawn character now works prop

* add try-get wrapper for preferences

* fix runtime in ignore

* remove cringe fix

* add proximity_checker nullable check

* add nullable check to camera shake

* fix roundend action runtime

* fix bad icon operation runtime

* fix respawn_character

Commit:
tweak(changeling): less blood more difficulty

* tweak(changeling): added steal identity objective, reduced the required amount of absorbed DNA
* prevent transformation trick

Nod404 committed:
tweak(smokables): reduces light power and range

PR #6137 
- Уменьшено свечение от горящих сигарет, сигар и т.д.

NefrNerf committed:
fix(armblades): fixes description of armblade and EMP gibbing fake armblades

PR #6016 
- Фейковый армблейд больше не гиббается от ЭМИ.
- Исправлено описание армблейда, убрана строчка про рефлект пуль.

Max1w2ll committed:
tweak(janitor): adds janitorial belt to janitor's closet

PR #5945

Doster-d committed:
feat(ai customs): adds example states for AI custom icons

PR #5292

hard-res3t committed:
tweak(Frontier): fixes assorted bugs

PR #5844
- У учёных есть доступ к своим техам.
- В кабинет АВД добавлен шреддер.
- В кабинет АВД добавлен планшет.
- Убран доступ шахтеров к докам карго шаттла.
- Добавлена кнопка для шторок в переговорную глав.
- Добавлен азот на базу воксов.

TobyThorne committed:
feat(sledgehammer): adds true space asshole's weapon

PR #5904

hard-res3t committed:
tweak(map): fixes assorted bugs

PR #5908 
- Починил доступ к генератору гравитации.
- Перенес одну стенку чтобы лестница не уходила в стену в техах у отбытия.
- Добавил воксам на базу азот.
- Починил невидимые деревья в снежном лесу на голодеке.
- Изменён спрайт голопеска.
- В кабинет АВД добавлен шреддер и планшет.

igorsaux committed:
tweak(rcd) fixes assorted bugs

PR #6152

* fix rcd

* Update code/game/objects/items/weapons/RCD.dm

Co-authored-by: HonkyDonky <59123747+HonkyDonky@users.noreply.github.com>

Co-authored-by: HonkyDonky <59123747+HonkyDonky@users.noreply.github.com>

igorsaux committed:
fix(runtime): unknown sound: science

NefrNerf committed:
fix(nullrod): fixes nullrod inability to decultify properly

* fix(nullrod): fixes nullrod inability to decultify properly
* delete(de-cultify): removes unused code

Commit:
fix(inflatables): borgs can no longer switching state remotely

PR #6160

Commit:
tweak(robots): borgs can deflect projectiles with energy swords

PR #6153

TobyThorne committed:
tweak(liver): adds safe toxic damage values

PR #5039 
- Теперь у печени есть запас токс-урона, который она может сдержать. Если запас не превышен - токсины будут постепенно выводиться из организма, не нанося урона. Если превышен - как и раньше, органы будут получать урон, начиная с печени и почек.

igorsaux committed:
Changelog 20/07/21 (#6162)

Nod404 committed:
tweak(graffiti): removes yoba-face

PR #6172

igorsaux committed:
fix(pedal generator): fixes runtime undefined variable /mob/living/silicon/robot/drone/var/handcuffed

PR #6171

igorsaux committed:
fix(story log): fixes ckey leak

PR #6170

Zert0X committed:
tweak(antags): adds vampire and thrall icons for antaghud

PR #6168

TobyThorne committed:
tweak(syringes/screwdrivers): syringes and screwdrivers are displayed vertically while in storages

PR #6163

TobyThorne committed:
tweak(gas stand): updates icons

PR #6161

malboec committed:
fix(machete): machetes now use a proper compile-time icon state

PR #6178

igorsaux committed:
tweak(TG UI): removes some tooltips

PR #6181 
- ...cause they cause lag

TobyThorne committed:
fix(crates): fixes multitool hacking

PR #6183 
- Исправлена невозможность взлома ящиков мультитулом.

Doster-d committed:
feat(medical machinery): assemble and disassemble medical equipment

* feat(medical machinery) add construction for medical machinery

* feat(optable) add optable deconstuction/construction

igorsaux committed:
optimization(RnD console): tiny optimization for a RnD console

Doster-d committed:
fix(borg): borgs now have sprite for open panel

PR #6199

Doster-d committed:
tweak(viruses): renames virus subsystem so we don't look like a bunch of morons

PR #6219

Oubifx committed:
fix(changeling): transform wear drop fix

* Transform fixes

igorsaux committed:
fix(global_vars): global_vars/Destroy() are working correctly

igorsaux committed:
remove(unused scripts): unused scripts removed

SerJo2 committed:
tweak(disposal): now you can climb inside in handcuffs

Nod404 committed:
remove(news): old news deleted

quardbreak committed:
add(toys): plush snail

Oubifx committed:
fix(rig healthanalyzer): fix health analyzer display output

* rig healthanalyzer fix

TobyThorne committed:
feat(tools): exclusive tools for CE

SerJo2 committed:
fix(name): Changed Happy Christmas to Merry Christmas

* Changed Happy Cristmas to Marry Cristmas

TobyThorne committed:
tweak(hear_sleep): fixes message formatting

PR #6221

Doster-d committed:
tweak(contracts): admin menu now display closed contracts

Doster-d committed:
tweak(contracts): empty contracts are no longer a thing

PR #6233

Zert0X committed:
conflicts

Zert0X committed:
Merge branch 'make-goats-great-again' of https://github.com/Zert0X/OnyxBay into make-goats-great-again

Zert0X committed:
Revert "Merge branch 'make-goats-great-again' of https://github.com/Zert0X/OnyxBay into make-goats-great-again"

This reverts commit 895ed2eab0d2358c7654e332a006b0c13517a610, reversing
changes made to 71c0ff53988ff0c204dc37f1da49ccebe991f959.

Zert0X commented:
![изображение](https://user-images.githubusercontent.com/42303535/127746063-c1d5bff6-e312-4345-8fbf-6be44b0e1c8a.png)
упс я дурной, я сломал...
